kern/preempttrap.c

```diff
@@ -1,3 +1,4 @@
+#include <linux/sched/task_stack.h>
```

Comment:
way way way down, `asm/processor.h` (indirectly imported through several layers by `uaccess.h` below) relies on a definition of `task_stack_page` from `linux/sched/task_stack.h`, which is does not import. So I just imported it here since it where the compiler was complaining.